### ISSUE TITLE
Add missing patches

### ISSUE THREAD

**Issue by ExE-Boss** [RPGHacker/SMW-Workspace#1]
Since you have released a total of [5 patches on SMW Central](https://www.smwcentral.net/?p=section&s=smwpatches&f[author]=RPG+Hacker), it might be a good idea to add all of them to this repository.

## Patches missing from this repository:
- [ ] [RPG-Styled HP and MP Counter](https://www.smwcentral.net/?p=section&a=details&id=6863)
- [ ] [Sonic Styled Powerdown Patch](https://www.smwcentral.net/?p=section&a=details&id=4231)
- [ ] [VWF Dialogues Patch](https://www.smwcentral.net/?p=section&a=details&id=12585)

## Patches already in this repository:
- [x] [Free 7F4000](https://www.smwcentral.net/?p=section&a=details&id=14237)
- [x] [Tolerance Timer](https://www.smwcentral.net/?p=section&a=details&id=14236)

**Comment by RPGHacker** [comment]
You're right, that's not a bad idea, just in case someone actually wants to contribute something to them. I think I'd want to rework them a bit before putting them here, though, mainly to get them consistent with each other. Might take a while before I get to it, but putting this one on my "to do" list.
